lib/matplotlib/figure.py

```diff
@@ -1226,12 +1230,14 @@ def _break_share_link(ax, grouper):
         if last_ax is not None:
             _reset_loc_form(last_ax.xaxis)
 
-    def clf(self, keep_observers=False):
+    def clf(self, keep_observers=False, disable_rc_fetch=True):
```

Comment:
Despite my comments in #7434 , I think a non-inverted kwarg (ex `reset_rcparams=False`) would be easier to understand.

Reply:
I agree. I'll change it.